Cargo.lock

```diff
@@ -55,14 +55,31 @@ dependencies = [
  "memchr 2.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "argon2rs"
```

Comment:
This appears to be added due to the `term` crate, whose author is looking for a new maintainer.

In a follow-up PR I'll take a look at switching to [termcolor](https://crates.io/crates/termcolor) which is both maintained and has significantly fewer dependencies (it actually appears to have no dependencies when using it on non-Windows platforms).